File: Plov/src/Rice.java

public class Rice {
	
	private boolean isDirty=true;
	private boolean ready;
	
	

	public boolean getDirty() {
		return isDirty;
	}
	
	public void setDirty(boolean d) {
		isDirty = d;
	}
	
	public boolean getReady() {
		return ready;
	}
	

	public void heat()
	{
		ready=true;
	}


}

File: Plov/Plov/Rice.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Plov
{
    class Rice
    {
        private bool dirty = true;
        private bool ready;

        public bool isDirty
        {
            set
            {
                dirty = value;
            }

            get
            {
                return dirty;
            }
        }

        public bool getReay
        {
            get
            {
                return ready;
            }
        }

        public void heat()
        {
            ready = true;
        }

    }
}
